ci/pipeline/determine_tests_to_run.py

```diff
@@ -15,7 +15,7 @@
     lint python cpp core_cpp java workflow compiled_graphs dashboard ray_client
     data serve ml tune train llm rllib rllib_gpu rllib_directly
     linux_wheels macos_wheels docker doc python_dependencies tools
-    release_tests compiled_python k8s_doc
+    release_tests compiled_python k8s_doc spark_on_ray
```

Comment:
Verified that this works by adding in my Pr - https://github.com/ray-project/ray/pull/53287/commits/7bbfb5403376a3a80c80381bcc17cd070419ac62